packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -507,7 +507,7 @@ export const Table = <
     {} as Record<string, string>,
   );
 
-  const areFiltersSet = allColumns.some((column) => !!column.filterValue);
+  const areFiltersSet = allColumns.some((column) => column.filterValue !== undefined && column.filterValue !== null && column.filterValue !== '');
```

Comment:
You can use `!=` to make one check for `undefined` and `null`.

Reply:
Updated this